eng/Packages.props

```diff
@@ -35,9 +35,12 @@
     <PackageReference Update="System.Reflection.TypeExtensions" Version="4.1.0" />
     <PackageReference Update="System.Resources.Extensions" Version="4.6.0" />
     <PackageReference Update="System.Resources.Writer" Version="4.0.0" />
+    <PackageReference Update="System.Runtime.InteropServices" Version="4.3.0" />
```

Comment:
@forgind do these versions have to be synced with nuget, the sdk, or any other tool in the .net core toolchain?

Reply:
@wli3? I don't think we have to worry about the .net core toolchain if we can insert into VS, but we should make sure that works. If someone has already used a different version, it's easiest to switch to theirs to avoid a mismatch if possible; if no one has used it, we'd have to add it, which can be tough.

Reply:
Version of System.Runtime.InteropServices (4.3.0) is the latest available, built in 2016 - all projects seem to reference that one.

System.Security.Cryptography.Pkcs and System.Security.Cryptography.Xml - versions 4.7.0 are from .NET Core 3.1 release (LTS).

I see references to these two versions for several other packages.